app/scenes/SceneBrowser.js

```diff
@@ -82,22 +82,16 @@ function sleep(millis, callback) {
     setTimeout(function() { callback(); }, millis);
 }
 
-SceneSceneBrowser.createCell = function(row_id, coloumn_id, data_id, data_name, thumbnail, title, info, info2, info_fill) {
+SceneSceneBrowser.createCell = function(row_id, coloumn_id, data_id, data_name, thumbnail, title, info, info2, info_fill, category) {
```

Comment:
`info_fill` is not used and should be removed.

Before it was used to not show info unless in GAMES mode, so by removing the if-else bellow you are always showing the info. Is this what you are intending?